src/main/java/ui/listpanel/ListPanel.java

```diff
@@ -235,40 +235,29 @@ private void setupKeyboardShortcuts() {
                 KeyPress.setLastKeyPressedCodeAndTime(event.getCode());
             }
             if (NEW_COMMENT.match(event)) {
-                if (KeyPress.isValidKeyCombination(GOTO_MODIFIER.getCode(), event.getCode())) {
-                    ui.getBrowserComponent().switchToTab(DISCUSSION_TAB);
-                } else {
-                    if (!ui.getBrowserComponent().isCurrentUrlIssue()) {
-                        openPageOfCurrentlySelectedIssue();
-                    } else if (!ui.getBrowserComponent().isCurrentUrlDiscussion()) {
-                        ui.getBrowserComponent().switchToTab(DISCUSSION_TAB);
+                if (!ui.getBrowserComponent().isCurrentUrlIssue()) {
+                    return;
```

Comment:
the go to modifier is the `G` button, btw. And it doesn't jump to comment box :)